app/AppKernel.php

```diff
@@ -21,6 +21,7 @@ public function registerBundles()
             $bundles[] = new Symfony\Bundle\DebugBundle\DebugBundle();
             $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
             $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
+            $bundles[] = new Symfony\Bundle\WebServerBundle\WebServerBundle();
```

Comment:
Why adding this package? 
It must be a dev package

>WebServerBundle provides commands for running applications using the PHP built-in web server. It simplifies your local development setup because you don't have to configure a proper web server such as Apache or Nginx to run your application.

Reply:
Yes, that's why I put it in `if (in_array($this->getEnvironment(), ['dev', 'test'], true)) {`

Reply:
Woops, my bad :roll_eyes: